src/amalgam/gen/neondot.c

```diff
@@ -1679,6 +1679,291 @@ void xnn_qd8_f32_qc8w_gemm_minmax_ukernel_4x8c4__neondot(
   } while (nc != 0);
 }
 
+void xnn_qd8_f32_qc4w_bl_gemm_minmax_ukernel_4x8c4__neondot(
```

Comment:
@fbarchard - FYI this doesn't spill on armv8. I haven't spend time to benchmark or even write a meta-kernel to codegen these yet.